packages/react/src/components/Slider/Slider.tsx

```diff
@@ -563,7 +718,7 @@ export default class Slider extends PureComponent {
       formatLabel = defaultFormatLabel,
       labelText,
       step,
-      stepMultiplier, // eslint-disable-line no-unused-vars
+      stepMultiplier: _stepMultiplier,
```

Comment:
what's the purpose of destructuring this prop here @tay1orjones ? doesn't seem like we're using it for rendering purposes, should we `delete other.stepMultiplier` as well in lines 731-732

Reply:
This prop is destructured off `this.props` so that it is not included in `...other`. `other` is spread onto a `div` and would error if this property was applied to it.

This is the only file in which the `delete` pattern is used, and it's quite old code (4+ years). By and large in the rest of the codebase we follow the pattern of destructuring out any props we don't want included in `other`, and turn off the unused variable eslint rule.

What's the purpose of adding this rename instead of retaining the `eslint-disable-line `?

Reply:
We have @typescript-eslint/no-unused-vars enabled, so unused vars must match /^_/u